### PR TITLE
Bugfix/#640 and #647

### PR DESCRIPTION
**Pull request by nvolodymyr**

### CODE REVIEW

File: src/app/component/auth/new-sign-up/new-sign-up.component.html [hunk 1]
```diff
@@ -188,7 +188,7 @@ <h2 class="subtitle-text">
                  class="sign-up-google-img"
             >
             <span class="sign-up-google-text">
-              {{ 'user.auth.sign-up.submit-with-google' | translate }}
+            {{'user.auth.sign-in.sign-in-with-google' | translate}}
```
Comment (RomanDeveloperAcc) [inline src/app/component/auth/new-sign-up/new-sign-up.component.html:191]: Please add one tab before brackets and spaces between brackets

File: src/app/component/auth/new-sign-up/new-sign-up.component.html [hunk 2]
```diff
@@ -188,7 +188,7 @@ <h2 class="subtitle-text">
                  class="sign-up-google-img"
```
Comment (nabermi2020) [inline src/app/component/auth/new-sign-up/new-sign-up.component.html:188]: image
You should add some fixes to the 'Forgot password' modal

Comment (nabermi2020) [reply src/app/component/auth/new-sign-up/new-sign-up.component.html:188]: Please, revert changes in the 'sign up' form controller

Comment (nvolodymyr) [reply src/app/component/auth/new-sign-up/new-sign-up.component.html:201]: fixed

**Pull request merged**